admin_interface/locale/fr/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,186 @@
+msgid "Header"
+msgstr "Bannière"
+
+#: admin.py:55
+msgid "Breadcrumbs / Module headers"
+msgstr "Fil d'ariane / en-têtes de module"
+
+#: admin.py:65
+msgid "Generic Links"
+msgstr "Liens génériques"
+
+#: admin.py:72
+msgid "Save Buttons"
+msgstr "Boutons sauvegarder"
+
+#: admin.py:80
+msgid "Delete Buttons"
+msgstr "Boutons supprimer"
+
+#: admin.py:88
+msgid "Related Modal"
+msgstr "Modale pour objet reliés"
+
+#: admin.py:97
+msgid "List Filter"
+msgstr "Filtre de liste"
+
+#: admin.py:101
+msgid "Recent Actions"
+msgstr "Actions récentes"
+
+#: apps.py:11
+msgid "Admin Interface"
+msgstr "Interface Admin"
+
+#: models.py:54
+msgid "name"
+msgstr "nom"
+
+#: models.py:57 models.py:220
+msgid "active"
+msgstr "actif"
+
+#: models.py:61 templates/admin/base_site.html:6
+#: templates/admin/base_site.html:66
+msgid "Django administration"
+msgstr "Administration Django"
+
+#: models.py:63
+msgid "title"
+msgstr "titre"
+
+#: models.py:69
+msgid "title color"
+msgstr "couleur du titre"
+
+#: models.py:72 models.py:87 models.py:108 models.py:253
+msgid "visible"
+msgstr "visible"
+
+#: models.py:77
+msgid "Leave blank to use the default Django logo"
+msgstr "Laisser vide pour utiliser le logo Django par défaut"
+
+#: models.py:78
+msgid "logo"
+msgstr "logo"
+
+#: models.py:84
+msgid "logo color"
+msgstr "couleur du logo"
+
+#: models.py:92
+msgid "(.ico|.png|.gif - 16x16|32x32 px)"
+msgstr "(.ico|.png|.gif - 16x16|32x32 px)"
+
+#: models.py:93
+msgid "favicon"
+msgstr "favicon"
+
+#: models.py:96
+msgid "Development"
+msgstr "Développement"
+
+#: models.py:97
+msgid "Testing"
+msgstr "Test"
+
+#: models.py:98
+msgid "Staging"
+msgstr "Pré-prod"
+
+#: models.py:99
+msgid "Production"
+msgstr "Production"
+
+#: models.py:105
+msgid "environment"
+msgstr "environment"
+
+#: models.py:115 models.py:140 models.py:181 models.py:200 models.py:226
+msgid "background color"
+msgstr "couleur d'arrière-plan"
+
+#: models.py:121 models.py:146 models.py:193 models.py:212 models.py:216
+msgid "text color"
+msgstr "couleur du texte"
+
+#: models.py:127 models.py:152 models.py:168
+msgid "link color"
+msgstr "couleur de lien"
+
+#: models.py:133 models.py:158 models.py:174
+msgid "link hover color"
+msgstr "couleur de lien au survol"
+
+#: models.py:161 models.py:246
+msgid "rounded corners"
+msgstr "coins arrondis"
+
+#: models.py:187 models.py:206
+msgid "background hover color"
+msgstr "couleur de fond au survol"
+
+#: models.py:243
+msgid "background opacity"
+msgstr "opacité de l'arrière-plan"
+
+#: models.py:250
+msgid "use dropdown"
+msgstr "utiliser un menu déroulant"
+
+#: models.py:262
+msgid "Theme"
+msgstr "Thème"
+
+#: models.py:263
+msgid "Themes"
+msgstr "Thèmes"
+
+#: templates/admin/filter.html:13
+#: templates/admin_interface/dropdown_filter.html:4
+#, python-format
+msgid " By %(filter_title)s "
+msgstr " Par %(filter_title)s "
+
+#: templates/admin/popup_response.html:3
+msgid "Popup closing..."
+msgstr "Fermeture de la modale..."
```

Comment:
I would like to review this string in context before this is merged!

Reply:
What do you mean?

Reply:
I mean that I am not sure of my own translation :sweat_smile: 
I will run my app and see if what I wrote makes sense when I see it in the real context.

Reply:
I think that this can be merged, any translation can be updated in future...

Reply:
![SHIP IT](https://media3.giphy.com/media/Hw8vYF4DNRCKY/giphy.gif?cid=3640f6095c33e59d5a6f35382e7324ce)

Reply:
I think packaging still needs a little update to make sure the `.mo` files are shipped and installed.

Reply:
`.mo` files are included in package, as you can see in `MANIFEST.in`
A good idea could be compile messages at packaging time, to be sure that `.mo` files are always updated each time a translation is updated or a new localization is added.

Reply:
I also see the `.mo` files in the wheel! :sparkles: :ferris_wheel: